setup.cfg

```diff
@@ -24,11 +24,19 @@ include_package_data = True
 python_requires = >=3.6
 install_requires =
     traitlets
+    entrypoints
+    importlib_resources
+    importlib_metadata
```

Comment:
You can also make this optional https://github.com/pypa/virtualenv/blob/adcf327ee5/setup.cfg#L46

Reply:
opionalized